AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_editor_utils.py

```diff
@@ -221,7 +221,7 @@ def timed_out(self):
         return time.time() > self.die_after
 
 
-screenshotsFolder = os.path.join(azlmbr.paths.products, "Screenshots")
+screenshotsFolder = os.path.join(azlmbr.paths.products.lower(), "screenshots")
```

Comment:
nit: all other os.path.join lines have the .lower() applied to the resulting join not in the middle.
```suggestion
screenshotsFolder = os.path.join(azlmbr.paths.products, "screenshots").lower()
```

Reply:
going to remove it for file paths but keep it for asset paths

Reply:
deleting `.lower()` calls from folders and files path but keep it for asset paths